projects/plugins/jetpack/extensions/plugins/seo/components/upsell.js

```diff
@@ -15,13 +16,19 @@ const UpsellNotice = ( { requiredPlan } ) => {
 
 	const buttonText = __( 'Upgrade now', 'jetpack' );
 	const supportUrl = 'https://wordpress.com/support/seo-tools/';
+	const postId = 120916;
+	const supportLinkTitle = __( 'Learn more about SEO feature.', 'jetpack' );
```

Comment:
I know the link currently says this already, but I am not sure if this is the best grammatical choice. I think it should say "Learn more about the SEO feature"